pkg/logs/input/docker/decoder.go

```diff
@@ -0,0 +1,56 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+// +build docker
+
+package docker
```

Comment:
Not fan of the packaging of this file, also, I think it should be `matcher.go` instead of `decoder.go`